src/routes/+layout.svelte

```diff
@@ -81,6 +82,10 @@
         saveReferrerAndUtmSource(page.url);
     });
 
+    $effect(() => {
+        applyTheme($currentTheme);
+    });
```

Comment:
we should rather subscribe to the theme store and update the theme like `currentTheme.subscribe(applyTheme);` so the change only happens when a theme is switched avoiding unnecessary function calls on any reactive change 👍